AutoCollection/CorrelationContextManager.ts

```diff
@@ -161,137 +161,6 @@ export class CorrelationContextManager {
         var nodeVer = process.versions.node.split(".");
         return parseInt(nodeVer[0]) > 3 || (parseInt(nodeVer[0]) > 2 && parseInt(nodeVer[1]) > 2);
```

Comment:
This version check needs to be bumped higher. [package.json of `cls-hooked`](https://github.com/Jeff-Lewis/cls-hooked/blob/master/package.json) says >= 4.7.0